packages/fuel-indexer/src/service.rs

```diff
@@ -286,3 +299,23 @@ async fn create_service_task(
         }
     }
 }
+
+async fn get_start_block(
+    conn: &mut IndexerConnection,
+    manifest: &Manifest,
+) -> Option<u64> {
+    let resumable = manifest.resumable.unwrap_or(false);
+    if resumable {
+        let last_block = queries::last_block_height_for_indexer(
+            conn,
+            &manifest.namespace,
+            &manifest.identifier,
+        )
+        .await
+        .unwrap_or(1) as u64;
```

Comment:
If `last_block_height_for_indexer` fails (for whatever reason) that's a legit error, and we should unwrap it as opposed to swallowing it with `.unwrap_or(1)` so maybe we can just 

```rust
        let last_block = queries::last_block_height_for_indexer(
            conn,
            &manifest.namespace,
            &manifest.identifier,
        )
        .await?;
```